### PR TITLE
Web Editor now available to Junior accounts [HC-1437]

### PR DESCRIPTION
**Pull request by seaxwi**

### CODE REVIEW

**Comment by Serjlee** [comment]
Be careful about @lukdog's note: https://arduino.slack.com/archives/C021U3KE9DW/p1701337827775019?thread_ts=1701275869.476339&cid=C021U3KE9DW
> I think it is not announced because it is not COPPA certified

It's probably better to hold any change on the public documentation unless it's decided otherwise

**Comment by Serjlee** [comment]
@seaxwi I suspect this shouldn't be amended until the use by Junior accounts is actually certified (c/c @lukdog )

**Comment by seaxwi** [comment]
@Serjlee @lukdog These changes are meant to reflect the fact that being signed in with a Junior account doesn't block access to the Web Editor. I don't see it as condoning or suggesting it's use by minors, or implying that it's certified. But unless the previous behavior is restored, the account type isn't relevant for troubleshooting, hence the changes to [content/Arduino Cloud/Web Editor/If-you-cant-access-the-Web-Editor.md](https://github.com/arduino/help-center-content/pull/307/files#diff-cd6565b113d3f738df3de75154146b8f0e10861d4f0a9a1f48e28bea02419b6b).

For [content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md](https://github.com/arduino/help-center-content/pull/307/files#diff-cd6565b113d3f738df3de75154146b8f0e10861d4f0a9a1f48e28bea02419b6b), we just removed this note:

> **Note:** The Web Editor is not currently supported for junior accounts.

If you like, we could restore a note stating more specifically:

> **Note:** The Web Editor is not currently certified for junior accounts.

File: content/Arduino Cloud/Cloud Editor/If-you-cant-access-the-Cloud-Editor.md [hunk 1]
```diff
@@ -13,6 +13,6 @@ When accessing the [Cloud Editor](https://support.arduino.cc/hc/en-us/articles/1
 
 Check the following:
 
-* Use a supported account type. The Cloud Editor is currently not available when signed in with a [junior account](https://support.arduino.cc/hc/en-us/articles/4839080453148-About-Arduino-accounts-for-minors).
+* If you are using a [junior account](https://support.arduino.cc/hc/en-us/articles/4839080453148-About-Arduino-accounts-for-minors#Junior-accounts), use the [dedicated sign-in page](app.arduino.cc/minors).
```
Comment (seaxwi) [inline content/Arduino Cloud/Cloud Editor/If-you-cant-access-the-Cloud-Editor.md:16]:
```suggestion
* If you are using a [junior account](https://support.arduino.cc/hc/en-us/articles/4839080453148-About-Arduino-accounts-for-minors#Junior-accounts), use the [dedicated sign-in page](https://app.arduino.cc/minors).
```

**Pull request merged**